pkg/webservice/webservice.go

```diff
@@ -81,7 +81,7 @@ type YResponseWriter struct {
 
 func (rw *YResponseWriter) Write(bytes []byte) (int, error) {
 	rw.body = bytes
-	if rw.statusCode == -1 {
+	if rw.statusCode == -1 || rw.statusCode == http.StatusOK {
```

Comment:
This seems to fix the problem we saw here, but this line could still cause issues.
For example, what if we get some other HTTP code defined in https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html. Such as `Accepted`, `Created`, etc.  This will still skip writing the body to the client-side, which is dangerous.

Reply:
Yes, correct. We should have taken a generic approach. Taken care now.